elements/pfe-modal/src/pfe-modal.js

```diff
@@ -83,6 +81,8 @@ class PfeModal extends PFElement {
   }
 
   _init() {
+    if (window.ShadyCSS) this._observer.disconnect();
```

Comment:
So we need to make sure that we disable mutation observers when altering the shadowdom?  Am I understanding that correctly?

Reply:
Exactly, otherwise IE11 can get stuck in an infinite loop and never load.